.orchestra/support/vscode-web/launch-vscode-web.py

```diff
@@ -0,0 +1,46 @@
+import webbrowser
```

Comment:
The Python standard library never stops to surprise me.

Reply:
`import antigravity`